README.md

```diff
@@ -1,66 +1,59 @@
 
 Design tokens provide a method to quickly and consistently implement the province's look and feel for digital services. They offer a set of standardized options to help developers style user interfaces in a consistent and future-proof way.
 
-* [Installable package on npm](https://www.npmjs.com/package/@bcgov/design-tokens)
-* [Source code](https://github.com/bcgov/design-system/tree/main/packages/design-tokens)
-* [Figma library](https://www.figma.com/community/file/1326994583954765832)
+- [Installable package on npm](https://www.npmjs.com/package/@bcgov/design-tokens)
+- [Source code](https://github.com/bcgov/design-system/tree/main/packages/design-tokens)
+- [Figma library](https://www.figma.com/community/file/1326994583954765832)
 
 ## Component library
 
 The component library will provide an inventory of reusable UI components, including:
 
-* Components in Figma
-* Reference implementations in React
-* Support for other languages and frameworks is currently out-of-scope.
+- Reusable components in Figma
+- Reference implementations in React
 
-The component library is currently in active development.
+Support for other languages and frameworks is currently out-of-scope.
 
-A [proof-of-concept/preview](https://www.npmjs.com/package/@bcgov/design-system-react-components) of the React library is published on npm.
+The component library is currently in active development. A [developer preview](https://www.npmjs.com/package/@bcgov/design-system-react-components) of the React library is published on npm.
 
 ## Contact
 
 The B.C. Design System is maintained by Government Digital Experience (GDX), a division of the Ministry of Citizens' Services. You can contact the design system team by email at [designsystem@gov.bc.ca](mailto:designsystem@gov.bc.ca), or [open an issue](https://github.com/bcgov/design-system/issues/new/choose).
 
 ## Legacy documentation
+
 > [!WARNING]
-> The previous B.C. government design system is not actively supported. These resources are provided for reference only. These resources will be officially deprecated and archived after the new BCDS component library is released.
+> The previous B.C. government design system is no longer supported and has been archived. These resources are provided for reference only.
 
-Documentation and components for the legacy design system are accessible in this repo and on DevHub:
+Documentation and components for the legacy design system are accessible in this repo and on Classic DevHub:
 
-* [Developer documentation](https://classic.developer.gov.bc.ca/Design-System/About-the-Design-System)
-* [Source code](https://github.com/bcgov/design-system/tree/main/components)
+- [Developer documentation](https://classic.developer.gov.bc.ca/Design-System/About-the-Design-System)
+- [Source code](https://github.com/bcgov/design-system/tree/main/components)
 
 ## License
 
     Copyright 2016 Province of British Columbia
 
-    Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Without these line breaks, this block overflows:

<img width="1046" alt="Screenshot 2024-04-23 at 12 58 30 PM" src="https://github.com/bcgov/design-system/assets/25143706/a4dcd8b2-49a6-4f7a-9dca-ae1157c32490">

Maybe we just take this summary license section out entirely and replace it with "See LICENSE file"?

Reply:
Weird! I think you're right — now that I think about it more closely, I don't know why this section is here at all. We have LICENSE.md accessible right here: 

<img width="912" alt="Screenshot 2024-04-24 at 10 10 49" src="https://github.com/bcgov/design-system/assets/135075821/7969f72b-882f-4b64-a71f-b09ee7db5b3d">

I've just made another commit (3f6742b) to remove that paragraph from README.md entirely.